documentation: ros2: run instructions for params files incorrect

**Describe the issue**
This issue covers 3 seperate potential problems in the "Run Instructions section" of the [README](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/README.md) on the ros2 branch:

- Linked [params.yml](https://github.com/LORD-MicroStrain/microstrain_inertial/blob/ros2/microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml) is missing.
- Steps to publish data from two devices simultaneously contains incomplete `params.yml` files which could be misleading.
- Running `ros2 launch microstrain_inertial_driver microstrain_launch.py node_name:=sensor_a_node namespace:=sensor_a params_file:="~/sensor_a_params.yml"` fails. with the following error:

```
user@8b30d4c51e22:~/your_workspace$ ros2 launch microstrain_inertial_driver microstrain_launch.py params_file:="~/sensor_a_params.yml"
[INFO] [launch]: All log files can be found below /home/user/.ros/log/2024-10-25-17-05-18-597342-8b30d4c51e22
[INFO] [launch]: Default logging verbosity is set to INFO
[WARNING] [launch_ros.actions.node]: Parameter file path is not a file: ~/sensor_a_params.yml
[INFO] [microstrain_inertial_driver_node-1]: process started with pid [6579]
[microstrain_inertial_driver_node-1] [INFO] [1729872318.690519498] [microstrain_inertial_driver]: Running microstrain_inertial_driver version: ros2-4.4.0
[microstrain_inertial_driver_node-1] [INFO] [1729872318.690600628] [microstrain_inertial_driver]: Using MIP SDK version: v1.0.0-283-g2c4cb6b
[microstrain_inertial_driver_node-1] [WARN] [1729872318.690605690] [microstrain_inertial_driver]: No relative position configured. We will not publish relative odometry or transforms.
[microstrain_inertial_driver_node-1] [WARN] [1729872318.690608143] [microstrain_inertial_driver]:   Please configure relative position to publish relative position data
[microstrain_inertial_driver_node-1] [INFO] [1729872318.690678571] [microstrain_inertial_driver]: Attempting to open serial port </dev/ttyACM0> at <115200>
[microstrain_inertial_driver_node-1] [ERROR] [1729872318.690696439] [microstrain_inertial_driver]: MIP SDK: Unable to open port (2): No such file or directory
[microstrain_inertial_driver_node-1] [ERROR] [1729872318.690720636] [microstrain_inertial_driver]: Failed to read configuration for node
[microstrain_inertial_driver_node-1] [ERROR] [1729872318.690723938] [microstrain_inertial_driver]: Failed to configure node base
[microstrain_inertial_driver_node-1] [FATAL] [1729872318.690727066] [microstrain_inertial_driver]: Failed to configure node
[ERROR] [microstrain_inertial_driver_node-1]: process has died [pid 6579, exit code 1, cmd '/home/user/your_workspace/install/microstrain_inertial_driver/lib/microstrain_inertial_driver/microstrain_inertial_driver_node --ros-args -r __node:=microstrain_inertial_driver -r __ns:=/ --params-file /tmp/launch_params_t30hz5c4 --params-file /tmp/launch_params_p8yuzbms'].
```

**Modifications**
Using absolute paths resolves the third problem

**Additional context**
These problems were encountered when trying to get started with microstrain_intertial with ROS 2.

The third problem relating to absolute paths could be a bug if relative paths for `params.yml` files is meant to be a supported feature.

Suggest correcting documentation in the short term and investigating a potential bug later.

--- a/README.md
+++ b/README.md
@@ -122,25 +122,29 @@ The node has some optional launch parameters that can be specified from the comm
 - `namespace` : namespace that the driver will run in. All services and publishers will be prepended with this, default: `/`
 - `node_name` : name of the driver, default: `microstrain_inertial_driver`
 - `debug`     : output debug logs, default: `false`
-- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](./microstrain_inertial_driver/microstrain_inertial_driver_common/config/params.yml), default: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
+- `params_file` : path to a parameter file to override the default parameters stored by default in: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
     
 #### Publish data from two devices simultaneously  
 
-1. Create the following files somewhere on your system (we will assume they are stored in the `~` directory):
+1. Create the following files somewhere on your system (we will assume they are stored in your home (`~`) directory):
     1. `~/sensor_a_params.yml` with the contents:
         ```yaml
-        port: /dev/ttyACM0
+        microstrain_inertial_driver:
+          ros__parameters:
+            port: /dev/ttyACM0
         ```
     2. `~/sensor_b_params.yml` with the contents:
         ```yaml
-        port: /dev/ttyACM1
+        microstrain_inertial_driver:
+          ros__parameters:
+            port: /dev/ttyACM1
         ```
 2. In two different terminals:
     ```bash    
-    ros2 launch microstrain_inertial_driver microstrain_launch.py node_name:=sensor_a_node namespace:=sensor_a params_file:="~/sensor_a_params.yml"
+    ros2 launch microstrain_inertial_driver microstrain_launch.py node_name:=sensor_a_node namespace:=sensor_a params_file:="$HOME/sensor_a_params.yml"
     ```
     ```bash    
-    ros2 launch microstrain_inertial_driver microstrain_launch.py node_name:=sensor_b_node namespace:=sensor_b params_file:="~/sensor_b_params.yml"
+    ros2 launch microstrain_inertial_driver microstrain_launch.py node_name:=sensor_b_node namespace:=sensor_b params_file:="$HOME/sensor_b_params.yml"
     ```
 
 This will launch two nodes that publish data to different namespaces:

--- a/README.md
+++ b/README.md
@@ -122,7 +122,12 @@ The node has some optional launch parameters that can be specified from the comm
 - `namespace` : namespace that the driver will run in. All services and publishers will be prepended with this, default: `/`
 - `node_name` : name of the driver, default: `microstrain_inertial_driver`
 - `debug`     : output debug logs, default: `false`
-- `params_file` : path to a parameter file to override the default parameters stored by default in: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
+- `params_file` : path to a parameter file to override the default parameters stored in [`params.yml`](https://github.com/LORD-MicroStrain/microstrain_inertial_driver_common/blob/main/config/params.yml), default: [`empty.yml`](./microstrain_inertial_driver/config/empty.yml)
+
+> [!NOTE]
+> The example [`params.yml`](https://github.com/LORD-MicroStrain/microstrain_inertial_driver_common/blob/main/config/params.yml) file is formatted to work with ROS and will not work if specified as the params_file argument in ROS2.
+>
+> If you want to override parameters for ROS2, start with [`empty.yml`](./microstrain_inertial_driver/config/empty.yml).
     
 #### Publish data from two devices simultaneously  
 

